### DISCUSSION TITLE
More stringent checks for files.

### DISCUSSION THREAD

**Discussion by Enlightnd** [Monitor221hz/Pandora-Behaviour-Engine-Plus#154] [Ideas]
Recently I had an issue where I updated and/or swapped out several mods, both functional and graphical and suddenly everything was T-posing.

Reverted everything, checked everything, T-posing persisted.

Days of trying various things later, I removed Pandora and reinstalled FNIS, which immediately told me that one of the HKX files of one of the mods I had was not compatible with SSE.

Somehow reverting the updates I had done still left this LE file that was put in by the author by accident.

But Pandora didn't warn me about any of this (neither did Nemesis for that matter), but FNIS did.

So it would seem there should be some checks built into Pandora, that FNIS always did on files, so that we have detection for clearly bad files that will 100% T-pose your install.

**Comment by lobotomy-x** [comment]
As a workaround you can extract this to your mods folder or to a specific mod and it will recursively scan for and automatically convert 32 bit HKX to 64 bit. Won't fix everything but it helps for some
[SEUpgrade.zip](https://github.com/Monitor144hz/Pandora-Behaviour-Engine-Plus/files/14729478/SEUpgrade.zip)
